ark/jsonschema/common.ts

```diff
@@ -0,0 +1,17 @@
+import { throwParseError } from "@ark/util"
+import { type JsonSchema, type Type, type } from "arktype"
+
+export const parseCommonJsonSchema = (
+	jsonSchema: JsonSchema
+): Type | undefined => {
+	if ("const" in jsonSchema) {
+		if ("enum" in jsonSchema) {
+			throwParseError(
+				"Provided JSON Schema cannot have both 'const' and 'enum' keywords."
```

Comment:
Generally for these sort of cases, the error is defined as its own const or writer function so it can easily be tested against the expected result.

I don't think we have a test for this yet anyways which is honestly totally fine. just figured I'd mentioned the pattern since its common in the rest of the repo for parse errors.

Reply:
Done in 429319de